README.md

```diff
@@ -23,27 +23,45 @@ you will need to have PHP installed to run it.
 Install it with your preferred package management tool
 (for example, on Ubuntu Linux you can run: `sudo apt-get install php`)
 
+The recommended way to install this library is with Composer.
+Composer is a dependency management tool for PHP
+that allows you to declare the dependencies your project needs
+and installs them into your project.
+
 Install Composer by following the instructions [here](https://getcomposer.org/doc/00-intro.md).
 
-Then, download Wikimate, and initialise it by running `composer install`
-(or `composer.bat install` if you're on Windows).
+Then, download Wikimate and initialise it by running:
```

Comment:
Perhaps in addition to the comma removal, reordering the sentence can make it clearer that the downloading is not a step to be performed manually:

```suggestion
Then, run the following command to download Wikimate and initialise it:
```

Also, it might be a nice touch to specify where this command should be run (e.g. "run the following command in your project's folder").

Reply:
Good idea, done.